common/locales/en/limited.json

```diff
@@ -33,16 +34,18 @@
     "skiSet": "Ski-sassin (Rogue)",
     "snowflakeSet": "Snowflake (Healer)",
     "yetiSet": "Yeti Tamer (Warrior)",
+    "toAndFromCard": "To: <%= toName %>, From: <%= fromName %>",
     "nyeCard": "New Year's Card",
+    "nyeCardExplanation": "For celebrating the new year together, you both receive the \"Auld Acquaintance\" badge!",
     "nyeCardNotes": "Send a New Year's card to a party member.",
     "seasonalItems": "Seasonal Items",
     "auldAcquaintance": "Auld Acquaintance",
     "auldAcquaintanceText": "Happy New Year! Sent or received <%= cards %> New Year's cards.",
-    "newYear0": "Happy New Year! May you slay many a bad Habit.",
-    "newYear1": "Happy New Year! May you reap many Rewards.",
-    "newYear2": "Happy New Year! May you earn many a Perfect Day.",
-    "newYear3": "Happy New Year! May your To-Do list stay short and sweet.",
-    "newYear4": "Happy New Year! May you not get attacked by a raging Hippogriff.",
+    "nye0": "\"Happy New Year! May you slay many a bad Habit.\"",
```

Comment:
Hmmmm.... we need to figure out why the escaped quotes are necessary. Something to do with the markdown, I expect.